examples/cvode/kokkos/cv_bruss_batched_kokkos.cpp

```diff
@@ -0,0 +1,408 @@
+/* -----------------------------------------------------------------------------
+ * Programmer(s): David J. Gardner and Cody J. Balos @ LLNL
+ * -----------------------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2002-2022, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ * -----------------------------------------------------------------------------
+ * The following is a simple example problem based off of ark_brusselator.c.
+ *
+ * We simulate a scenario where a set of independent ODEs are batched together
+ * to form a larger system. Each independent ODE system has 3 components,
+ * Y = [u,v,w], satisfying the equations,
+ *    du/dt = a - (w+1)*u + v*u^2
+ *    dv/dt = w*u - v*u^2
+ *    dw/dt = (b-w)/ep - w*u
+ * for t in the interval [0.0, 10.0], with initial conditions Y0 = [u0,v0,w0].
+ * The problem is stiff. We have 3 different testing scenarios:
+ *
+ * Reactor 0:  u0=3.9,  v0=1.1,  w0=2.8,  a=1.2,  b=2.5,  ep=1.0e-5
+ *    Here, all three components exhibit a rapid transient change
+ *    during the first 0.2 time units, followed by a slow and
+ *    smooth evolution.
+ *
+ * Reactor 1:  u0=3,  v0=3,  w0=3.5,  a=0.5,  b=3,  ep=5.0e-4
+ *    Here, all components undergo very rapid initial transients
+ *    during the first 0.3 time units, and all then proceed very
+ *    smoothly for the remainder of the simulation.
+ *
+ * Reactor 2:  u0=1.2,  v0=3.1,  w0=3,  a=1,  b=3.5,  ep=5.0e-6
+ *    Here, w experiences a fast initial transient, jumping 0.5
+ *    within a few steps.  All values proceed smoothly until
+ *    around t=6.5, when both u and v undergo a sharp transition,
+ *    with u increaseing from around 0.5 to 5 and v decreasing
+ *    from around 6 to 1 in less than 0.5 time units.  After this
+ *    transition, both u and v continue to evolve somewhat
+ *    rapidly for another 1.4 time units, and finish off smoothly.
+ *
+ * This program solves the problem with the BDF method, Newton iteration, a
+ * user-supplied Jacobian routine, and since the grouping of the independent
+ * systems results in a block diagonal linear system, with the KOKKOS
+ * SUNLinearSolver which supports batched iterative methods. 100 outputs are
+ * printed at equal intervals, and run statistics are printed at the end.
+ *
+ * The program takes three optional arguments, the number of independent ODE
+ * systems (i.e. number of batches), the linear solver type
+ * (KOKKOS batched LU, non-batched GMRES with the Jacobian computed by
+ * difference quotients, or non-batched GMRES with analytical Jacobian), and
+ * the test type (uniform_1, uniform_2, uniform_3, or random).
```

Comment:
```suggestion
 * (KOKKOS batched LU, or non-batched GMRES with the Jacobian computed by
 * difference quotients), and the test type (uniform_1, uniform_2, uniform_3, or random).
```
There are commenting inconsistencies in `solver_type`.  The paragraph above indicates 3 options (KOKKOS batched LU, non-batched GMRES with the Jacobian computed by difference quotients, and non-batched GMRES with analytical Jacobian).  However, the comments here and the code itself only support 2 options (no GMRES with analytical Jacobian).